dev/drivers/scripts/plots/analyses/jevs_analyses_grid2obs_plots.sh

```diff
@@ -8,16 +8,20 @@
 #PBS -l select=1:ncpus=1:mem=10GB
 #PBS -l debug=true
 
+set -x
+
 export model=evs
 
 export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
 
 source $HOMEevs/versions/run.ver
 
+export evs_ver=v1.0.0
+evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
```

Comment:
To my understanding, we don't need a 2d version of evs_ver. compath.py will return a 2d version.